modules/ROOT/pages/storage.adoc

```diff
@@ -335,7 +335,7 @@ storage:
 
 This example creates a swap partition spanning all of the `sdb` device, creates a swap area on it, and creates a systemd swap unit so the swap area is enabled on boot.
 
-NOTE: Using `with_mount_unit: true` with `format: swap` requires Butane spec version `1.4.0-experimental`.  After spec 1.4.0 is stabilized, version `1.4.0-experimental` will no longer be accepted by Butane and configs that use it will need to be updated.
+NOTE: Using `with_mount_unit: true` with `format: swap` requires Butane spec version `1.4.0-experimental`.  After spec 1.4.0 is stabilized, version `1.4.0-experimental` will no longer be accepted by Butane and the corresponding Ignition config version will no longer be accepted by Ignition.
```

Comment:
```suggestion
NOTE: Using `with_mount_unit: true` with `format: swap` requires Butane spec version `1.4.0-experimental`.  After spec 1.4.0 is stabilized, version `1.4.0-experimental` will no longer be accepted by Butane and the corresponding Ignition config version will no longer be accepted by Ignition and configs will need to be updated to replace `1.4.0-experimental` with `1.4.0`.
```

Optional addition to make the required future change blatantly obvious.

Of course, it is possible that it won't be that simple (i.e. the spec changes for some reason and the experimental version isn't directly promoted), but I think it's fair to gloss over that here.

Reply:
If we're documenting some part of the experimental spec in official docs, I'd assume we're fairly confident that we'll stabilize that part as-is.

Updated to add some more words.